.gitignore

```diff
@@ -23,4 +23,7 @@ besspin-ci.log
 *.asm
 *.o
 *.bin
-besspin/cyberPhys/hacking/j1939hack/main.h
\ No newline at end of file
+besspin/cyberPhys/hacking/j1939hack/main.h
+besspin/cyberPhys/ui/can-display/package-lock.json
```

Comment:
These are generated so you can reproduce builds.  They could be useful later, but Iḿ not sure even the deployment model we have now how we would track them.

Reply:
Easy to add them back if they are important!